packages/widget/src/types/token.ts

```diff
@@ -3,4 +3,6 @@ import type { TokenAmount as SDKTokenAmount } from '@lifi/sdk'
 export interface TokenAmount extends SDKTokenAmount {
   featured?: boolean
   popular?: boolean
+  marketCapUSD?: number | null
+  volumeUSD24H?: number | null
```

Comment:
The latest SDK version includes the TokenExtended type, which we can inherit from here